ground/src/main/java/com/google/android/ground/ui/home/HomeScreenFragment.kt

```diff
@@ -99,6 +100,9 @@ class HomeScreenFragment :
     navHeader.findViewById<TextView>(R.id.switch_survey_button).setOnClickListener {
       homeScreenViewModel.showSurveySelector()
     }
+    navHeader.findViewById<ShapeableImageView>(R.id.ground_logo).setOnClickListener {
+      showSignOutConfirmationDialogs()
+    }
```

Comment:
The click for signout confirmation dialog should be bound to the profile image and not app logo.